client/ayon_core/pipeline/stagingdir.py

```diff
@@ -0,0 +1,218 @@
+from ayon_core.lib import Logger, filter_profiles, StringTemplate
+from ayon_core.settings import get_project_settings
+from ayon_core.pipeline.template_data import get_template_data
+
+from .anatomy import Anatomy
+from .tempdir import get_temp_dir
+
+STAGING_DIR_TEMPLATES = "staging"
+
+
+def get_staging_dir_config(
+    host_name,
+    project_name,
+    task_type,
+    task_name,
+    product_type,
+    product_name,
```

Comment:
I would move host name after product information to match most of other pipeline functions we have.
```suggestion
    project_name,
    task_type,
    task_name,
    product_type,
    product_name,
    host_name,
```